src/org/thoughtcrime/securesms/ConversationListAdapter.java

```diff
@@ -195,7 +189,12 @@ interface ItemClickListener {
   }
 
   void changeData(@Nullable DcChatlist chatlist) {
-    dcChatlist = chatlist == null ? new DcChatlist(0, 0) : chatlist;
+    if (chatlist == null) {
+      dcChatlist =  new DcChatlist(0, 0);
+    } else {
+      dcChatlist = chatlist;
+      dcContext = DcHelper.getAccounts(context).getAccount(chatlist.getAccountId());
```

Comment:
is it more optimal to check if chatlist.getAccountId() == previous acccountId instead of always calling `getAccount()`?